src/components/tipRecords/TipPreview.vue

```diff
@@ -23,7 +23,7 @@
             class="tip__amount"
             @click.stop
           >
-            <TipInput :tip="tip" />
+            <slot />
```

Comment:
TipPreview is always shown for a usual tip with TipInput for that tip, so why you have extracted TipInput?